core/generated-sources/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/ColumnsSelectionDsl.kt

```diff
@@ -3868,24 +4085,18 @@ public interface ColumnsSelectionDsl<out T> : ColumnSelectionDsl<T>, SingleColum
 
     // region ranges
 
-    @Suppress("UNCHECKED_CAST")
     public fun <C> ColumnSet<C>.cols(range: IntRange): ColumnSet<C> =
-        transformWithContext {
-            dataFrameOf(it)
-                .asColumnGroup()
-                .cols(range)
-                .resolve(this) as List<ColumnWithPath<C>>
-        }
+        colsInternal(range) as ColumnSet<C>
 
     public operator fun <C> ColumnSet<C>.get(range: IntRange): ColumnSet<C> = cols(range)
 
-    public fun SingleColumn<AnyRow>.cols(range: IntRange): ColumnSet<*> =
-        transform { it.flatMap { it.children().subList(range.first, range.last + 1) } }
+    public fun SingleColumn<*>.cols(range: IntRange): ColumnSet<*> =
+        colsInternal(range)
 
     /**
      *
      */
-    public operator fun SingleColumn<AnyRow>.get(range: IntRange): ColumnSet<*> = cols(range)
+    public operator fun SingleColumn<*>.get(range: IntRange): ColumnSet<*> = cols(range)
```

Comment:
Why? I think SingleColumn<AnyRow> is a ColumnGroup and this operations make sense, but not for SingleColumn<*>

Reply:
Well yes, but then it breaks when you 
```kotlin
df.select { it["name"][0..5] }
```
since it doesn't know the type of the single column you select that way.

Also, I'm gonna close this PR in favor of dfs-rename3, just waiting for Tolya to answer.